server/src/main/java/org/elasticsearch/cluster/coordination/ClusterBootstrapService.java

```diff
@@ -62,6 +62,9 @@ public class ClusterBootstrapService implements Coordinator.PeerFinderListener {
 
     static final String BOOTSTRAP_PLACEHOLDER_PREFIX = "{bootstrap-placeholder}-";
 
+    // extract the docs link here to force loading of the ReferenceDocs class, validating early in startup that all its links are defined.
```

Comment:
If we need to ensure ReferenceDocs is initialized early, let's do it explicitly in a class that is loaded early, either Elasticsearch or Node? Also, we can use the following to force static init:

```
    try {
        MethodHandles.publicLookup().ensureInitialized(ReferenceDocs.class);
    } catch (IllegalAccessException unexpected) {
        throw new AssertionError(unexpected);
    }
```

Reply:
Ah nice ok I'll do that instead - see 8fbdb55f3fc.